### PR TITLE
Fix Issue 1537

### PR DESCRIPTION
**Pull request by yamamashakaa**
After using Export to CSV button  The Submit Button in Patient_list.php behaves like the Export to CSV Button and don't submit values.
Solution: Correct the condition in the if statement and set the value of
form_csv to false when clicking submit.
Issue: 1537

### CODE REVIEW

**Comment by Ngai-E** [comment]
This works fine thanks.
One suggestion try creating a separate branch when working on a fix or feature then send a PR for it other than messing up with you masters branch.

File: library/report_functions.php [hunk 1]
```diff
@@ -145,7 +145,7 @@ function showSubmitPrintButtons($form_csvexport='') {
                     <div style='margin-left:5px'>
                         <a href='#' class='css_button cp-submit' onclick='$(\"#form_refresh\").attr(\"value\",\"true\"); $(\"#theform\").submit();'>
                             <span>";
-                                echo xlt('Submit');
+                         echo xlt('Submit');
```
Comment (muarachmann) [inline library/report_functions.php]: Hmmmm should this be here?

File: interface/usergroup/addrbook_edit.php [hunk 2]
```diff
@@ -252,7 +252,7 @@ function typeSelect(a) {
     invalue('form_notes')         . ", " .
     invalue('form_abook_type')    . " "  .
    ")");
-
+
```
Comment (muarachmann) [inline interface/usergroup/addrbook_edit.php]: Well surely the editor doing stuffs

File: interface/reports/patient_list.php [hunk 3]
```diff
@@ -31,9 +31,10 @@
 require_once "reports_controllers/PatientListController.php";
 
 // In the case of CSV export only, a download will be forced.
-if ($_POST['form_csvexport']) {
-  csvexport('patient_list'); // CSV headers. (TRK)
+if ($_POST['form_csvexport'] && !$_POST['form_refresh']) {
+    csvexport('patient_list'); // CSV headers. (TRK)
 } else {
+     $_POST['form_csvexport']=false;
```
Comment (muarachmann) [inline interface/reports/patient_list.php]: LGTM

**Comment by yamamashakaa** [comment]
Happy that it got merged. Thanks for the advise, yes next time I will create a branch isntead of playing with the master

**Pull request merged**